### PR TITLE
K4

### PR DESCRIPTION
**Pull request by kinuesantos**

### CODE REVIEW

File: lesson4/public/index28.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title></title>
+</head>
+<body>
+  <form action= '/send'>
+  Name:<input type='text' name="a1"/>
+  Comment:<input type='text' name="a2"/>
```
Comment (saumya665) [inline lesson4/public/index28.html]: change the id of name and comment. make them a bit realistic instead of a1 and a2.

Comment (kinuesantos) [reply lesson4/public/index28.html]: done!

**Review by MadhuBeesetty — APPROVED** [review]
next time a better file names, so that it will be easier to review your lessons when you want them later.

**Review by pkdotson — APPROVED** [review]

File: lesson4/backend28.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const fs = require('fs');
+const express= require('express')
+const app = express();
+app.listen(3618);
+app.use(express.static('public'));
+
+app.get('/send',(req,res)=>{
+  const firstdata = req.query.thenames;
+  const seconddata = req.query.thecomments;
+  fs.writeFile('/home/kinue/garageScript/lesson4/public/storing.txt', firstdata + seconddata)
```
Comment (allopez7) [inline lesson4/backend28.js]: put space and colon between firstdata +': '+ seconddata +'\n'. 
OR 
Optional: const nameComment = `${req.query.name}: ${req.query.comment}\n`

Comment (pkdotson) [inline lesson4/backend28.js]: firstdata + " " + seconddata

**Review by pkdotson — APPROVED** [review]

File: lesson4/backend28.js (hunk 2 above)
Comment (kinuesantos) [reply lesson4/backend28.js]: done! thanks

**Pull request merged**